### ISSUE TITLE
Details error related to Gradient.lua

### ISSUE THREAD

**Issue by brandonlee781** [Toxicom/toxiui#44]
### Description
I get this error every time I enter combat. The error log seems to point towards Details. In fact if I change to a different details profile there error stops happening. I can run the Toxi installer and get a fresh details profile and that fixes the error for a while, but eventually I log back in and this error pops up again.

### Screenshots
![Screenshot 2023-02-19 141845](https://user-images.githubusercontent.com/11919542/219973089-645602c9-aeb8-44d2-ae41-241af12d64cc.png)

### Notes
Lua Error:
```Message: ...Ons/ElvUI_ToxiUI/Modules/Themes/Gradients/Update.lua:35: attempt to index local 'fgMap' (a nil value)
Time: Sun Feb 19 13:52:34 2023
Count: 1
Stack: ...Ons/ElvUI_ToxiUI/Modules/Themes/Gradients/Update.lua:35: attempt to index local 'fgMap' (a nil value)
[string "=[C]"]: ?
[string "@Interface/AddOns/ElvUI_ToxiUI/Modules/Themes/Gradients/Update.lua"]:35: in function `SetGradientColors'
[string "@Interface/AddOns/ElvUI_ToxiUI/Modules/Skins/Details/Gradients.lua"]:57: in function <...Ons/ElvUI_ToxiUI/Modules/Skins/Details/Gradients.lua:37>
[string "=(tail call)"]: ?
[string "=(tail call)"]: ?
[string "@Interface/AddOns/Details/frames/window_main.lua"]:4937: in function <Interface/AddOns/Details/frames/window_main.lua:4695>
[string "=[C]"]: in function `InstanceRefreshRows'
[string "@Interface/AddOns/ElvUI_ToxiUI/Modules/Skins/Details/Gradients.lua"]:16: in function `RefreshDetails'
[string "@Interface/AddOns/ElvUI_ToxiUI/Modules/Skins/Details/Gradients.lua"]:89: in function `SettingsUpdate'
[string "@Interface/AddOns/ElvUI_ToxiUI/Modules/Skins/Details/Gradients.lua"]:135: in function `Enable'
[string "@Interface/AddOns/ElvUI_ToxiUI/Modules/Skins/Details/Gradients.lua"]:154: in function <...Ons/ElvUI_ToxiUI/Modules/Skins/Details/Gradients.lua:153>

Locals: 
```
### To reproduce
1. Reloading the ui and any time I enter combat the error will pup up

### Acceptance criteria

-   [ ]

**Comment by Toxicom** [comment]
Which details windows are active? Damage/Healing etc.

**Comment by brandonlee781** [comment]
Damage and healing. Changing the bar texture or background texture also throws the errors in either windows options.

**Comment by brandonlee781** [comment]
Here is the Bars:General options for Damage done, but it's the same for Healing done as well:
![Screenshot 2023-02-19 150435](https://user-images.githubusercontent.com/11919542/219975139-8599c76d-1037-4348-8cbe-5ca9eafd97cb.png)

**Comment by Toxicom** [comment]
Is this still happening? Do we know how to reliably reproduce this?

**Comment by brandonlee781** [comment]
I don't know, I stopped using Toxui a while ago, and have taken a break from WoW since then.

**Comment by Toxicom** [comment]
We've done some fixes post-6.1.5 for gradient mode, so this might be fixed. Closing due to inactivity, re-open if needed 👌
